README.md

```diff
@@ -10,7 +10,7 @@ And add the following to your .emacs file:
 ;; shexc mode
 ;;
 
-(add-to-list 'load-path "{path}/shexc-mode.el")
+(add-to-list 'load-path "{folder that contains shexc-mode.el}")
```

Comment:
Tx for the PR.

Maybe `{path to shexc-mode.el}` to make it clear that they should include the file as well?

Reply:
The name you propose makes more sense but having the name of the file didn't work for me. Maybe it is because I'm using Doom Emacs?